pkg/expr/identity_test.go

```diff
@@ -143,38 +145,73 @@ func TestEvalPredicate(t *testing.T) {
 		attrs *mixerpb.Attributes
 		in    string
 		out   interface{}
-		err   bool
+		err   string
```

Comment:
We do want to know what expr thought the error was. If it is unresolved attribute, or invalid expression etc.